tests/Makefile.am

```diff
@@ -219,3 +223,9 @@ AM_TESTS_ENVIRONMENT =								\
 	PYTHONPATH=$(top_builddir)/python:$$PYTHONPATH				\
 	LD_LIBRARY_PATH=$(top_builddir)/liblouis/.libs:$$LD_LIBRARY_PATH	\
 	PATH=$(top_builddir)/tools:$$PATH
+
+if MINGW
+# FIXME: is there a way to prepend to LOG_COMPILE instead of to overwrite it?
+LOG_COMPILE = wine32 $(LOG_COMPILER) $(AM_LOG_FLAGS) $(LOG_FLAGS)
+YAML_LOG_COMPILE = wine32 $(YAML_LOG_COMPILER) $(AM_YAML_LOG_FLAGS) $(YAML_LOG_FLAGS)
```

Comment:
Isn't this supposed to be called `YAML_LOG_COMPILER` (according to the [Automake manual](https://www.gnu.org/software/automake/manual/automake.html#Parallel-Test-Harness))